install-cockroachdb.md

```diff
@@ -204,7 +204,7 @@ $(document).ready(function(){
 
     <p>The first time you run <code class="highlighter-rouge">make install</code>, it can take awhile to download and install various dependencies.</p>
 
-    <div class="bs-callout bs-callout-success">This step builds a binary for the latest beta release. To get access to <a href="develop-branch.html">features not yet included in a beta release</a>, you can build a binary from the <code>develop</code> branch instead. Simply leave out the <code>git checkout {{site.data.strings.version}}</code> step above.</div>
+    <div class="bs-callout bs-callout-success">This step builds a binary for the latest beta release. To get access to <a href="develop-branch.html">features not yet included in a beta release</a>, you can build a binary from the <code>develop</code> branch instead. Simply leave out the <code>git checkout latest</code> step above.</div>
```

Comment:
I don't think `git checkout latest` will work (we could make it work if we wanted to, but it would be a change to our release process).